<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonDemoGUIClass</class>
 <widget class="QMainWindow" name="SkeletonDemoGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkeletonDemoGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>942</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="MyPainter" name="Painter" native="true"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="menuLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="openImageButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>Обработать картинку</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveImageButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>Сохранить картинку</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshSkeletonButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>Обновить скелет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePolyButton">
         <property name="text">
          <string>Сохранить полигоны</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Параметр стрижки:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pruning">
         <property name="maximum">
          <number>100000000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Игнорировать площадь меньше:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="area">
         <property name="maximum">
          <number>100000000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="invertCheckBox">
         <property name="text">
          <string>Инвертировать картинку</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="filenameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Имя файла:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Время:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="imageCB">
         <property name="text">
          <string>Изображение</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bonesCB">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ребра</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="circlesCB">
         <property name="text">
          <string>Круги</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="contoursCB">
         <property name="text">
          <string>Контуры</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyPainter</class>
   <extends>QWidget</extends>
   <header>mypainter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
